docsource/020_required_knowledge.rst

```diff
@@ -1,4 +1,30 @@
 Required Knowledge
 ==================
 
-TODO : write here prerequisites.
\ No newline at end of file
+To Use Openupgrade
+------------------
+
+* you should be able to launch an instance of Odoo on your local PC,
+  or on your server, **for each version of your migration**.
+  For example, if you're migrating from version 12.0 to 16.0,
+  you should be able to launch Odoo versions 13.0, 14.0, 15.0 and 16.0.
```

Comment:
```suggestion
  you should be able to launch Odoo versions 13.0, 14.0, 15.0 and 16.0.
  After last migration finalizes, remember to run update=all without OpenUpgrade to let odoo delete deprecated records.
```

Reply:
thanks for the proposal, but that is not the purpose of this page. this page explain the knowledge you have to have to make a migration, not the way to do a migration.